components/AssetTabContent/AssetTabContentSectionContainer.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components"
+
+export const AssetTabContentSectionContainer = styled.section`
+  &:not(:last-child) {
+    margin-bottom: 1.5rem;
+  }
+
+  h2 {
+    margin-bottom: 0.5rem;
```

Comment:
why is that?

Reply:
the way I see it a given component shouldn't set its spacings, since that often times depends on the context in which it's used. but if we were to set the rules for spacing for each component, and especially for typography, the rule set can become extremely large.